### PR TITLE
docs: devtools throttling deprecation notice

### PR DESCRIPTION
**Pull request by connorjclark**
This doc will added via a link next to the throttling option in DevTools.

CL: https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/1885347

### CODE REVIEW

**Comment by brendankenny** [comment]
Would this be better on devsite or in a "What's New In DevTools" post? Seems weird to document here for whatever reason

**Comment by paulirish** [comment]
nah it's great here. just perfect. really the finest location. best place on earth.

**Comment by paulirish** [comment]
but i'm actually gonna merge this lil thing into the throttling.md doc and it'll be even better then.

File: docs/devtools-throttling-deprecation.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# DevTools Throttling Deprecation Notice
+
+Starting with Chrome 80, Lighthouse within Chrome DevTools is deprecating the throttling configuration. To wit:
```
Comment (brendankenny) [inline docs/devtools-throttling-deprecation.md]:
```suggestion
Starting with Chrome 80, Lighthouse within Chrome DevTools is deprecating the throttling configuration.
```

File: docs/devtools-throttling-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+1. `Simulated Throttling` will become the only option
+1. For awhile, you may also use `Applied Throttling` by unchecking the `Simulated Throttling` checkbox
```
Comment (brendankenny) [inline docs/devtools-throttling-deprecation.md]: fwiw a radio button seems less confusing for this. If "Applied Throttling" is unchecked I would assume it's not throttled, personally

Comment (brendankenny) [inline docs/devtools-throttling-deprecation.md]: should be "a while" but a different possibility might be something like
```suggestion
1. `Applied Throttling` will still be available by unchecking the `Simulated Throttling` checkbox
```

File: docs/devtools-throttling-deprecation.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+1. `No Throttling` is removed
```
Comment (brendankenny) [inline docs/devtools-throttling-deprecation.md]:
```suggestion
1. `No Throttling` will no longer be an option
```

**Comment by paulirish** [comment]
https://github.com/GoogleChrome/lighthouse/blob/dt-deprecation/docs/throttling.md#audits-panel-throttling-settings

File: docs/throttling.md [hunk 4]
```diff
@@ -14,13 +14,26 @@ These exact figures are used as [Lighthouse's throttling default](https://github
...
 
-Lighthouse, by default, uses simulated throttling as it provides both quick evaluation and minimized variance. However, some may want to experiment with more accurate throttling...
+Lighthouse, by default, uses simulated throttling as it provides both quick evaluation and minimized variance. However, some may want to experiment with more accurate throttling... [Learn more about these throttling types and how they behave in in different scenarios](https://www.debugbear.com/blog/network-throttling-methods).
+
+## Audits Panel Throttling Settings
+
+In Chrome 79 and earlier, you could choose between [the throttling types](#types-of-throttling) of Simulated, Applied, and none.
+
+Starting with Chrome 80, the Audits panel is simplifying the throttling configuration. To wit:
```
Comment (brendankenny) [inline docs/throttling.md]:
```suggestion
Starting with Chrome 80, the Audits panel is simplifying the throttling configuration.
```

File: docs/throttling.md [hunk 5]
```diff
@@ -14,13 +14,26 @@ These exact figures are used as [Lighthouse's throttling default](https://github
...
+
+1. _Simulated throttling_ becomes the default setting. This matches the setup of PageSpeed Insights and the Lighthouse CLI default, so you should see more consistent results.
+1. _No throttling_ is removed as it leads to innacurate scoring and misleading metric results.
+1. Within the Audits panel settings, you can uncheck the _Simulated throttling_ checkbox to use _Applied throttling_.
```
Comment (brendankenny) [inline docs/throttling.md]: fwiw a radio button seems significantly less confusing for this. If "Simulated Throttling" is unchecked I would assume it's not throttled, personally

File: docs/throttling.md [hunk 6]
```diff
@@ -14,13 +14,26 @@ These exact figures are used as [Lighthouse's throttling default](https://github
...
+1. **_Proxy-level_** tools do not affect UDP data, so they're decent, but not ideal.
 1. **_Packet-level_** throttling tools are able to make the most accurate network simulation. While this approach can model real network conditions most effectively, it also can introduce [more variance](https://docs.google.com/document/d/1BqtL-nG53rxWOI5RO0pItSRPowZVnYJ_gBEQCJ5EeUE/edit) than request-level or simulated throttling. [WebPageTest uses](https://github.com/WPO-Foundation/wptagent/blob/master/docs/remote_trafficshaping.md) packet-level throttling.
 
-Lighthouse, by default, uses simulated throttling as it provides both quick evaluation and minimized variance. However, some may want to experiment with more accurate throttling...
+Lighthouse, by default, uses simulated throttling as it provides both quick evaluation and minimized variance. However, some may want to experiment with more accurate throttling... [Learn more about these throttling types and how they behave in in different scenarios](https://www.debugbear.com/blog/network-throttling-methods).
+
+## Audits Panel Throttling Settings
```
Comment (brendankenny) [inline docs/throttling.md]: maybe call out **DevTools** Audits Panel clearly here

File: docs/throttling.md [hunk 7]
```diff
@@ -14,13 +14,26 @@ These exact figures are used as [Lighthouse's throttling default](https://github
...
-1. **_Proxy-level_** throttling tools do not affect UDP data, so they're decent, but not ideal.
+1. **_Request-level throttling_** , also referred to as **_Applied throttling_** in the Audits panel or _`devtools` throttling_ in Lighthouse configuration, is how throttling is implemented with Chrome DevTools. In real mobile connectivity, latency affects things at the packet level rather than the request level. As a result, this throttling isn't highly accurate. It also has a few more downsides that are summarized in [Network Throttling & Chrome - status](https://docs.google.com/document/d/1TwWLaLAfnBfbk5_ZzpGXegPapCIfyzT4MWuZgspKUAQ/edit). The TLDR: while it's a [decent approximation](https://docs.google.com/document/d/10lfVdS1iDWCRKQXPfbxEn4Or99D64mvNlugP1AQuFlE/edit), it's not a sufficient model of a slow connection. The [multipliers used in Lighthouse](https://github.com/GoogleChrome/lighthouse/blob/3be483287a530fb560c843b7299ef9cfe91ce1cc/lighthouse-core/lib/emulation.js#L33-L39) attempt to correct for the differences.
+1. **_Proxy-level_** tools do not affect UDP data, so they're decent, but not ideal.
```
Comment (patrickhulce) [inline docs/throttling.md]: feels weird to remove it just from here, `Proxy-level tools` but `Packet-level throttling tools`

what motivated the removal?

File: docs/throttling.md [hunk 8]
```diff
@@ -14,13 +14,26 @@ These exact figures are used as [Lighthouse's throttling default](https://github
...
-Lighthouse, by default, uses simulated throttling as it provides both quick evaluation and minimized variance. However, some may want to experiment with more accurate throttling...
+Lighthouse, by default, uses simulated throttling as it provides both quick evaluation and minimized variance. However, some may want to experiment with more accurate throttling... [Learn more about these throttling types and how they behave in in different scenarios](https://www.debugbear.com/blog/network-throttling-methods).
+
+## Audits Panel Throttling Settings
+
+In Chrome 79 and earlier, you could choose between [the throttling types](#types-of-throttling) of Simulated, Applied, and none.
+
+Starting with Chrome 80, the Audits panel is simplifying the throttling configuration. To wit:
+
+1. _Simulated throttling_ becomes the default setting. This matches the setup of PageSpeed Insights and the Lighthouse CLI default, so you should see more consistent results.
```
Comment (patrickhulce) [inline docs/throttling.md]: wasn't it already the default for new users? are we just removing the stickiness?

File: docs/throttling.md [hunk 9]
```diff
@@ -14,13 +14,26 @@ These exact figures are used as [Lighthouse's throttling default](https://github
...
+1. _No throttling_ is removed as it leads to innacurate scoring and misleading metric results.
+1. Within the Audits panel settings, you can uncheck the _Simulated throttling_ checkbox to use _Applied throttling_.
+
+For the moment, we are keeping _Applied throttling_ option available for users of the [`View Trace` button](https://developers.google.com/web/updates/2018/04/devtools#traces). Under applied throttling, the trace matches the metrics values, whereas under Simulated things do not currently match up.
+
+We plan to improve the experience of viewing a trace under simulated throttling. At that point, the _Applied throttling_ option will be removed from the Audits panel. Of course, CLI users can still control the exact [configuration](../#cli-options) of throttling.
```
Comment (brendankenny) [inline docs/throttling.md]: I guess somehow DevTools isn't for power users anymore :P

File: docs/throttling.md (hunk 7 above)
Comment (paulirish) [reply docs/throttling.md]: mistake

File: docs/throttling.md (hunk 8 above)
Comment (paulirish) [reply docs/throttling.md]: okay yah. reworded.

Comment (connorjclark) [reply docs/throttling.md]: I think calling out that it will become the _only_ option is what's important.

File: docs/throttling.md (hunk 5 above)
Comment (paulirish) [reply docs/throttling.md]: it's super weird. i agree. so let's get a reasonable lantern trace generated and then we can drop this silliness.

Comment (brendankenny) [reply docs/throttling.md]: > it's super weird. i agree. so let's get a reasonable lantern trace generated and then we can drop this silliness.
> -- former UX engineer Paul Irish

:P

**Pull request merged**